### DISCUSSION TITLE
Authentication Issue - TypeError: Deepgram.__init__() takes 1 positional argument but 2 were given

### DISCUSSION THREAD

**Discussion by jameshorton2486** [deepgram/community#553] [General help] [answered]
[Colab Notebook.txt](https://github.com/deepgram/community/files/14041879/Colab.Notebook.txt)

I created this notebook in Google Colab a few months ago.  I used to be able to run this without any problems.  I have spent several hours trying to figure out what I am doing wrong without any success.  If someone could help me, I would really appreciate it.  There are 5 different sections in the notebook.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment]
My guess is that you aren't aware of the major SDK update. The Python SDK has gone through a major update (`v2` to `v3`) which require code changes. If you run the code, you should get the following error message:

```
            FATAL ERROR:
            You are attempting to instantiate a Deepgram object, which is no longer a class in version 3 of this SDK.

            To fix this issue:
                1. You need to revert to the previous version 2 of the SDK: pip install deepgram-sdk==2.12.0
                2. or, update your application's code to use version 3 of this SDK. See the README for more information.

            Things to consider:

                - This Version 3 of the SDK requires Python 3.10 or higher.
                  Older versions (3.9 and lower) of Python are nearing end-of-life: https://devguide.python.org/versions/
                  Understand the risks of using a version of Python nearing EOL.

                - Version 2 of the SDK will receive maintenance updates in the form of security fixes only.
                  No new features will be added to version 2 of the SDK.
```

**Comment by davidvonthenen** [comment, marked as answer]
I created this PR to address this problem:
https://github.com/deepgram/deepgram-python-sdk/pull/271

It should be available in the next release.
